pyaci/core.py

```diff
@@ -161,13 +161,21 @@ def _rootApi(self):
 
 
 class Node(Api):
-    def __init__(self, url, session=None, verify=False, disableWarnings=True, timeout=None):
+    def __init__(self, url, session=None, verify=False, disableWarnings=True, timeout=None, aciMetaFilePath=None):
         super(Node, self).__init__()
         self._url = url
         if session is not None:
             self._session = session
         else:
             self._session = requests.session()
+
+        if aciMetaFilePath:
```

Comment:
Please change this to `if aciMetaFilePath is not None`.

Reply:
Done !